modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java

```diff
@@ -259,10 +259,12 @@ public void processOpts() {
             setPackageName("openapi");
         }
 
+        // If no version is provided in additional properties, version from API specification is used.
+        // If none of them is provided then fallback to default version
         if (additionalProperties.containsKey(CodegenConstants.PACKAGE_VERSION)) {
             setPackageVersion((String) additionalProperties.get(CodegenConstants.PACKAGE_VERSION));
-        } else {
-            setPackageVersion("1.0.0");
```

Comment:
can we still keep the else clause as the default package version setting?

Reply:
I mean, *we can*, but it is unnecessary. The default version is already set [above](https://github.com/OpenAPITools/openapi-generator/blob/16c9d4047d953fd09694c1affe517871979f3c39/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustClientCodegen.java#L52) in the same file.

Although I personally don't mind and can revert if you want.

Reply:
ok. let's go with your approach.